package.json

```diff
@@ -117,6 +117,42 @@
         "command": "flatpak-vscode.run",
         "linux": "ctrl+alt+R"
       }
+    ],
+    "jsonValidation": [
+      {
+        "fileMatch": [
+          "/*.*.*.json",
+          "/*.*.*.*.json",
+          "/*.*.*.*.*.json"
+        ],
+        "url": "./manifest-schema.json"
+      }
+    ],
+    "yamlValidation": [
+      {
+        "fileMatch": "/*.*.*.yaml",
+        "url": "./manifest-schema.json"
+      },
+      {
+        "fileMatch": "/*.*.*.*.yaml",
+        "url": "./manifest-schema.json"
+      },
+      {
+        "fileMatch": "/*.*.*.*.*.yaml",
+        "url": "./manifest-schema.json"
+      },
+      {
+        "fileMatch": "/*.*.*.yml",
+        "url": "./manifest-schema.json"
+      },
+      {
+        "fileMatch": "/*.*.*.*.yml",
+        "url": "./manifest-schema.json"
+      },
+      {
+        "fileMatch": "/*.*.*.*.*.yml",
+        "url": "./manifest-schema.json"
+      }
```

Comment:
Can't this be made an array like the json one?

Reply:
Unfortunately not (at least it didn't work on my machine)
Probably also something for a feature request.

Reply:
Ah, no worries then